changes/tasks.py

```diff
@@ -524,6 +560,17 @@ def check_momconnect_change_language(self, data_fields, change):
         else:
             return []
 
+    def check_momconnect_change_msisdn(self, data_fields, change):
+        """
+        Ensures that a new msisdn is provided, and that it is a valid msisdn.
+        """
+        if "msisdn" not in data_fields:
```

Comment:
The issue with a `previous_default` field is that there could be multiples (there could be multiple defaults, or no default so we have to include all numbers). Then, for the defaults, do we also want to include all the additional data?

I think we should either just store everything under `msisdn`, or we store all the details of everything under `msisdn` that we consider "default" (either default, or everything if there's no default).